docs/user-guide/docker-plugin-drivers.md

```diff
@@ -13,3 +13,28 @@ The Docker Vmdk volume driver supports provisioning and managing docker volumes
 The Docker Photon Volume driver supports provisioning and managing docker volumes on a Photon platform consisting of a cluster of ESX hosts managed via a Photon controller instance. Docker volumes are created and managed entirely via the open Photon platform API via the Photon controller.
 
 The Docker Volume plugin can support either or both types of volumes, as required, on a given Docker host.
+
+## Configuring the Docker Volume Plugin
+The docker volume plugin is designed to load run time options and values from a json configuration file (default /etc/docker-volume-vsphere.conf) on the host. The user can also provide a configuration file, via the "--config" option, specifying the full path of the file. The file contains the values for run time options used by the plugin. Options that are currently recognized include the below set. Options provided on the command line will override those in the configuration file.
+
+# Options for the photon volume driver
+Target    - URL at which to contact the Photon Controller
+Project   - project ID in Photon to which the docker host belongs
```

Comment:
Can we add commands or link to Photon documentation on how to to get "Project" and "Host" details. Also add sample value?